test/e2e/framework/framework.go

```diff
@@ -120,6 +128,10 @@ func deleteResource(client *ClientSet, resource string) error {
 		err = client.Core.Services(consoleapi.OpenShiftConsoleNamespace).Delete(consoleapi.OpenShiftConsoleServiceName, &metav1.DeleteOptions{})
 	case "Route":
 		err = client.Routes.Routes(consoleapi.OpenShiftConsoleNamespace).Delete(consoleapi.OpenShiftConsoleRouteName, &metav1.DeleteOptions{})
+	case "OCCLIDownloads":
```

Comment:
Same here, wish we didn't need these two internal custom strings `OCCLIDownloads` and `ODOCLIDownloads`.